lib/fluent/plugin/out_kafka2.rb

```diff
@@ -218,12 +222,22 @@ def write(chunk)
         end
       end
     rescue Exception => e
+      ignore = false
+      if ignore_exceptions.size > 0
+        ignore_exceptions.each { | ignore_exception |
+          ignore = true if e.class.name == ignore_exception
+        }
+      end
+
       log.warn "Send exception occurred: #{e}"
       log.warn "Exception Backtrace : #{e.backtrace.join("\n")}"
+      log.warn "Exception ignored in tag : #{tag}" if ignore
       # For safety, refresh client and its producers
       refresh_client(false)
+      # raise UnrecoverableError for backup ignored exception chunk
+      raise Fluent::UnrecoverableError if exception_backup
```

Comment:
`if ignore && exception_backup`?
Recoverable errors are also ignored?

Reply:
`if ignore && exception_backup` (even if Fluent::UnrecoverableError defined in conf) Recoverable error will not ignored.

what do you think about which is the best way to backup chunk? (I want to know which log makes exception)